src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -273,8 +291,11 @@ internal static void CheckForSevereException(Exception e)
             }
             finally
             {
-                TelemetryAPI.ReportExitTelemetry(s_theConsoleHost);
-                s_theConsoleHost.Dispose();
+                if (s_theConsoleHost != null)
```

Comment:
This always confuses me.  Isn't this the console std out?  If so then again I presume it is not available when no process console is available.  In any case it seems that trace output would be better for debugging a rare case of command line errors in server mode.